services/organization.js

```diff
@@ -13,44 +10,7 @@ class Organization {
       return result?.length > 0 ? result[0] : null;
   }
 
-  /**
-   * List Organizations API Interface.
-   *
-   * - `ADMIN` and `ORG_OWNER can call this API
-   * - `ORG_OWNER` is limited to only their organization
-   *
-   * @api
-   * @param {Object} params Endpoint parameters
-   * @param {{ cookie: Object, userInfo: Object }} context request context
-   * @returns {Promise<Object[]>} An array of Organizations
-   */
-  async listOrganizationsAPI(params, context) {
-    if (!context?.userInfo?.email || !context?.userInfo?.IDP) {
-        throw new Error(ERROR.NOT_LOGGED_IN)
-    }
-    if (context?.userInfo?.role !== USER.ROLES.ADMIN && context?.userInfo.role !== USER.ROLES.ORG_OWNER) {
-        throw new Error(ERROR.INVALID_ROLE);
-    }
-    if (context.userInfo.role === USER.ROLES.ORG_OWNER && !context?.userInfo?.organization?.orgID) {
-        throw new Error(ERROR.NO_ORG_ASSIGNED);
-    }
-
-    const filters = {};
-    if (context?.userInfo?.role === USER.ROLES.ORG_OWNER) {
-        filters["_id"] = context?.userInfo?.organization?.orgID;
-    }
-
-    return this.listOrganizations(filters);
-  }
-
-  /**
-   * List organizations by an optional set of filters
-   *
-   * @typedef {Object<string, any>} Filters K:V pairs of filters
-   * @param {Filters} [filters] Filters to apply to the query
-   * @returns {Promise<Object[]>} An array of Organizations
-   */
-  async listOrganizations(filters = []) {
+  async listOrganizations(filters) {
```

Comment:
The separation of `listOrganizations` and `listOrganizationsAPI` is new code added that we definitely don't want reverted.